parsec/core/gui/main_window.py

```diff
@@ -387,25 +388,22 @@ def showMaximized(self, skip_dialogs=False):
             )
 
         # For each parsec update
-        if self.config.gui_last_version != PARSEC_VERSION:
+        if self.config.gui_last_version and self.config.gui_last_version != PARSEC_VERSION:
 
-            # Ask for acrobat reader workaround
-            if (
-                platform.system() == "Windows"
-                and win_registry.is_acrobat_reader_dc_present()
-                and win_registry.get_acrobat_app_container_enabled()
-            ):
-                r = ask_question(
-                    self,
-                    _("TEXT_ACROBAT_CONTAINERS_DISABLE_TITLE"),
-                    _("TEXT_ACROBAT_CONTAINERS_DISABLE_INSTRUCTIONS"),
-                    [_("ACTION_ACROBAT_CONTAINERS_DISABLE_ACCEPT"), _("ACTION_NO")],
-                )
-                if r == _("ACTION_ACROBAT_CONTAINERS_DISABLE_ACCEPT"):
-                    win_registry.set_acrobat_app_container_enabled(False)
+            # Update from parsec `<1.13` to `>=1.13`
+            if LooseVersion(self.config.gui_last_version) < "1.13":
+
+                # Revert the acrobat reader workaround
+                if (
+                    platform.system() == "Windows"
+                    and win_registry.is_acrobat_reader_dc_present()
+                    and not win_registry.get_acrobat_app_container_enabled()
+                ):
+                    win_registry.del_acrobat_app_container_enabled()
```

Comment:
We don't have that many users yet so removing the workaround when updating from under `1.13` to above `1.13` seems like an acceptable thing to do.

> What is the behavior if we uninstall Parsec instead of updating it ?

The workaround will stay in place until the new parsec installation.